From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Wed, 9 Oct 2019 23:43:28 +0000
Subject: ANDROID: gki_defconfig: Add qcom pcie options to gki_defconfig

The qcom PCIE support isn't able to be added as a module, so I
propose enabling CONFIG_PCIE_QCOM and CONFIG_PCIEPORTBUS in
the gki_defconfig.

RAS falls out, as when CONFIG_PCIEPORTBUS=y PCIEAER defaults
to =y, which selects RAS

For consistency, this also adds CONFIG_PCIEPORTBUS=y to x86
gki_defconfig

Change-Id: I04941e17f660dd13b2ef2ab3150acae3ce670b19
Bug: 146449535
Signed-off-by: John Stultz <john.stultz@linaro.org>
---
 arch/arm64/configs/gki_defconfig | 3 ++-
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index a7d499362bd5..b677e11c93b6 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -210,7 +210,9 @@ CONFIG_MAC80211=y
 # CONFIG_MAC80211_RC_MINSTREL is not set
 CONFIG_RFKILL=y
 CONFIG_PCI=y
+CONFIG_PCIEPORTBUS=y
 CONFIG_PCI_HOST_GENERIC=y
+CONFIG_PCIE_QCOM=y
 CONFIG_PCIE_KIRIN=y
 CONFIG_FW_LOADER_USER_HELPER=y
 CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
@@ -395,7 +397,6 @@ CONFIG_IIO=y
 CONFIG_PWM=y
 CONFIG_QCOM_PDC=y
 CONFIG_GENERIC_PHY=y
-CONFIG_RAS=y
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
 CONFIG_INTERCONNECT=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 93c684a0cec0..905ee839c125 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -183,6 +183,7 @@ CONFIG_MAC80211=y
 # CONFIG_MAC80211_RC_MINSTREL is not set
 CONFIG_RFKILL=y
 CONFIG_PCI=y
+CONFIG_PCIEPORTBUS=y
 CONFIG_FW_LOADER_USER_HELPER=y
 CONFIG_FW_LOADER_USER_HELPER_FALLBACK=y
 # CONFIG_FW_CACHE is not set
